backends/bmv2/simple_switch/simpleSwitch.cpp

```diff
@@ -790,6 +791,32 @@ void ExternConverter_action_selector::convertExternInstance(
     ctxt->action_profiles->append(action_profile);
 }
 
+Util::IJson* ExternConverter_log_msg::convertExternFunction(
+    UNUSED ConversionContext* ctxt, UNUSED const P4::ExternFunction* ef,
```

Comment:
These parameters are actually not unused, so please remove the UNUSED qualifier on the ones which are used.